coq/coq-syntax.el

```diff
@@ -1321,13 +1321,10 @@ Very similar to `coq-omit-proof-admit-command', but without the dot."
     "\\<Error:")
   "A list of regexps matching coq error headers.")
 
-(defvar coq--raw-error-regexp (coq--regexp-alt-list coq--error-header-re-list))
+(defvar coq-error-regexp (coq--regexp-alt-list coq--error-header-re-list))
```

Comment:
`coq--prefix-not-regexp` is not needed any  more, is it? Do you want to delete it together with the comment in front of it?

Reply:
Forgot that. will do.